homeassistant/components/alarm_control_panel/ifttt.py

```diff
@@ -0,0 +1,118 @@
+"""
+Interfaces with alarm control panels that have to be controlled through IFTTT.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/alarm_control_panel.ifttt_securitysystem/
+"""
+import logging
+import voluptuous as vol
+
+import homeassistant.components.alarm_control_panel as alarm
+from homeassistant.components.alarm_control_panel import PLATFORM_SCHEMA
+from homeassistant.const import (
+    STATE_ALARM_ARMED_AWAY, SERVICE_ALARM_ARM_AWAY,
+    STATE_ALARM_ARMED_HOME, SERVICE_ALARM_ARM_HOME,
+    STATE_ALARM_ARMED_NIGHT, SERVICE_ALARM_ARM_NIGHT,
+    STATE_ALARM_DISARMED, SERVICE_ALARM_DISARM,
+    CONF_NAME, CONF_CODE)
+import homeassistant.helpers.config_validation as cv
+
+REQUIREMENTS = ['requests==2.18.4', 'pyfttt==0.3']
```

Comment:
Do not add requests. It's already a core dependency of Home Assistant. Also move the requests import to the top of the file.

Reply:
Actually, thinking of it. This platform should depend on the `ifttt` component instead of asking for the key itself. The `ifttt` component should be added via `DEPENDENCIES` and should share the key via `hass.data[DOMAIN]`

Reply:
Thanks for the advice! Unfortunately, I miss the point how I can get the key specified in the ifttt component. Could you please help me out?

Reply:
`ifttt` component already requires the `[key](https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/ifttt.py#L23)`. You can use `DEPENDENCIES = ['ifttt']` to use the IFTTT component as a dependency and not have user to enter the key again here. There are several components that do that, e.g., Arlo, Abode.

Reply:
Thanks for the suggestions! It made me realize I don't even need the key. When I have the ifttt component as a dependency, I can just call the component's trigger service

Reply:
Exactly...that is the point 👍